week2.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<p>The package start-up messages can be a bit confusing especially because they only show up once. The <code>suppressMessages</code>  function works well for this: <code>suppressMessages(library(tidyverse))</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>